### ISSUE TITLE
Usage (Meteor 1.3)

### ISSUE THREAD

**Issue by solace** [vazco/meteor-universe-i18n#16]
Hi,

I have a feeling I have missed something terribly obvious.

Using Meteor 1.3 (8) 

`import {_i18n} from 'meteor/universe:i18n';` - OK

`_i18n.__('a.b.c');`

with yml file `en-au.i18n.yml` (the only one there, and locale is returning as en-AU in the browser) (or `en-AU`, or `en`).

yml file contents:

```
a:
  b:
    c: "Some string"
```

Is returning 'a.b.c' instead of the corresponding "Some string".

Have also tried just using unnested values and it still just returns the key.

However `_i18n.addTranslations` works fine. :/ What have I neglected to notice?

**Comment by cristo-rabani** [comment]
Hello,
I must point that this package wasn't fully tested with meteor 1.3 but we know only that package ecmascript steals json files for self.Maybe yml files have some problems too.

So, I have questions for you:
Did You tried check what local is currently `_i18n.getLocale()`
Can you see your strings here:  `_i18n._translations.en-AU` ?

Have you seen some errors in console? 
Or in network tab?
You have this language added to bundle or is downloaded on demand.
As a default option only `en-us and en` are bundled for clients (if you not change this by env)
Only bundled languages are available from the beginning (so, your default/master locale should be always bundled by you) locales that are downloaded on demand, they must be downloaded so they will be after while.  


You can also try build your project with environment variable  `UNIVERSE_I18N_LOCALES=all` and checks.

**Comment by solace** [comment]
Hi,

I was setting the locale earlier and `getLocale` was returning the correct information.  There were no errors in the console, or on the server, or in the network tab.  It was added to the package installed and imported where used.

I had to remove this functionality so that I could continue working on features, and will test `_i18n._translations.en-AU` when I make another attempt at i18n in the next week.

Thanks for providing diagnostic assistance. If it's okay, I'll keep this open until I can check that last bit.
